core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -147,6 +157,7 @@ public void setStrategiesConfig(Map<String, Object> strategiesConfig) {
         configItems.add(new ConfigurationItem("debug", SMART_TESTING_DEBUG, false));
         configItems.add(new ConfigurationItem("autocorrect", SMART_TESTING_AUTOCORRECT, false));
         configItems.add(new ConfigurationItem("customStrategies", SMART_TESTING_CUSTOM_STRATEGIES_PATTERN));
+        configItems.add(new ConfigurationItem("customProviders", SMART_TESTING_CUSTOM_PROVIDERS, new String[0]));
```

Comment:
This got me thinking - we should document it as part of developer guide. For now, we only have test-bed. Can you create an issue for it? I believe it would be good to explain the concept and have a short list of things required to implement in order to have the configuration wired together.

Reply:
I tried to write some documentation about it, but I do agree that I was also missing some "developer part". I'll create the issue

Reply:
https://github.com/arquillian/smart-testing/issues/289